providers/NMSReflectionProvider/src/main/java/net/ess3/nms/refl/providers/ReflPersistentDataProvider.java

```diff
@@ -0,0 +1,125 @@
+package net.ess3.nms.refl.providers;
+
+import net.ess3.nms.refl.ReflUtil;
+import net.ess3.provider.PersistentDataProvider;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.plugin.Plugin;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Field;
+import java.util.Locale;
+
+public class ReflPersistentDataProvider implements PersistentDataProvider {
```

Comment:
```suggestion
/**
 * Stores persistent data on 1.18-1.13 in a manner that's consistent with PDC on 1.14+ to enable seamless upgrades.
 */
public class ReflPersistentDataProvider implements PersistentDataProvider {
```

(is this even correct comment syntax)

Reply:
yes but it's very ugly